payments-core/src/main/java/com/stripe/android/payments/core/injection/AuthenticationModule.kt

```diff
@@ -1,82 +1,39 @@
 package com.stripe.android.payments.core.injection
 
-import com.stripe.android.Logger
-import com.stripe.android.PaymentAuthConfig
-import com.stripe.android.PaymentBrowserAuthStarter
-import com.stripe.android.PaymentRelayStarter
-import com.stripe.android.networking.AnalyticsRequestExecutor
-import com.stripe.android.networking.AnalyticsRequestFactory
-import com.stripe.android.networking.StripeRepository
-import com.stripe.android.payments.Stripe3ds2CompletionStarter
-import com.stripe.android.payments.core.authentication.NoOpIntentAuthenticator
-import com.stripe.android.payments.core.authentication.Stripe3DS2Authenticator
+import com.stripe.android.model.StripeIntent.NextActionData
+import com.stripe.android.payments.core.authentication.IntentAuthenticator
+import com.stripe.android.payments.core.authentication.OxxoAuthenticator
 import com.stripe.android.payments.core.authentication.WebIntentAuthenticator
-import com.stripe.android.stripe3ds2.service.StripeThreeDs2Service
-import com.stripe.android.stripe3ds2.transaction.MessageVersionRegistry
-import com.stripe.android.view.AuthActivityStarterHost
+import dagger.Binds
 import dagger.Module
-import dagger.Provides
-import javax.inject.Singleton
-import kotlin.coroutines.CoroutineContext
+import dagger.multibindings.IntoMap
 
 /**
- * [Module] for com.stripe.android.payments.core.authentication.
- *
- * All non-daggerized dependencies are passed in as constructor parameters for now, a parameter
- * will be removed when it's daggerized and introduced to the dagger graph.
+ * Provides mappings between [NextActionData] and [IntentAuthenticator] provided by payment SDK.
  */
 @Module
-internal class AuthenticationModule(
-    private val stripeRepository: StripeRepository,
```

Comment:
instead of passing them as constructor params of `AuthenticationModule`, all dependencies required to build the dagger graph is provided through `AuthenticationComponent.Builder` with `@BindsInstance`, this way all these dependencies will be available in all `@Inject`ed constrcutors, make it possible to directly declare injection from classes like `WebIntentAuthenticator`